syntax = "proto3";

message TextMessage {
  string id = 1;
  uint64 timestamp = 2;
  string content = 3;
}

service EchoService {
  rpc Echo (TextMessage) returns (TextMessage);
  rpc EchoMany (stream TextMessage) returns (stream TextMessage);
}
